packages/vue/src/components/Overlays.ts

```diff
@@ -19,7 +19,7 @@ import { defineCustomElement as defineIonPopoverCustomElement } from '@ionic/cor
 
 import { defineOverlayContainer } from '../vue-component-lib/overlays';
 
-export const IonActionSheet = /*@__PURE__*/ defineOverlayContainer<JSX.IonActionSheet>('ion-action-sheet', defineIonActionSheetCustomElement, ['animated', 'backdropDismiss', 'buttons', 'cssClass', 'enterAnimation', 'header', 'htmlAttributes', 'keyboardClose', 'leaveAnimation', 'mode', 'subHeader', 'translucent'], actionSheetController);
+export const IonActionSheet = /*@__PURE__*/ defineOverlayContainer<JSX.IonActionSheet>('ion-action-sheet', defineIonActionSheetCustomElement, ['animated', 'backdropDismiss', 'buttons', 'cssClass', 'enterAnimation', 'header', 'htmlAttributes', 'isOpen', 'keyboardClose', 'leaveAnimation', 'mode', 'subHeader', 'translucent', 'trigger'], actionSheetController);
```

Comment:
You can't pass the controller if you want this call `createInlineComponent`:  https://github.com/ionic-team/ionic-framework/blob/c0f4381745bb0e373e8a1833756fd3f2a49dbf4a/packages/vue/src/vue-component-lib/overlays.ts#L171

Also looks like the same is true for `IonLoading` from the previous PR.